packages/dds/map/src/test/mocha/summary.spec.ts

```diff
@@ -0,0 +1,186 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+import { existsSync, readFileSync, writeFileSync } from "node:fs";
+import path from "node:path";
+import {
+	MockContainerRuntimeFactory,
+	MockFluidDataStoreRuntime,
+	MockStorage,
+} from "@fluidframework/test-runtime-utils";
+import { convertSummaryTreeToITree } from "@fluidframework/runtime-utils";
+import { DirectoryFactory, IDirectory, SharedDirectory } from "../..";
+
+const fileBase = path.join(__dirname, "../../../src/test/mocha/snapshots/");
+
+interface TestScenario {
+	only?: boolean;
+	skip?: boolean;
+	name: string;
+	runScenario: () => SharedDirectory;
+}
+
+function serialize(directory: SharedDirectory): string {
+	const summaryTree = directory.getAttachSummary().summary;
+	const snapshotTree = convertSummaryTreeToITree(summaryTree);
+	return JSON.stringify(snapshotTree, undefined, 1);
+}
+
+function assertEquivalentDirectories(first: IDirectory, second: IDirectory): void {
```

Comment:
this and below looks like equivalent functionality to `packages\dds\map\src\test\mocha\directoryEquivalenceUtils.ts`. Can we import from there instead?